src/vanilla.ts

```diff
@@ -57,16 +64,10 @@ export default function create<TState extends State>(
   ) => {
     let currentSlice: StateSlice = selector(state)
     function listenerToAdd() {
-      // Selector or equality function could throw but we don't want to stop
-      // the listener from being called.
-      // https://github.com/react-spring/zustand/pull/37
-      try {
-        const newStateSlice = selector(state)
-        if (!equalityFn(currentSlice, newStateSlice)) {
-          listener((currentSlice = newStateSlice))
-        }
-      } catch (error) {
-        listener(null, error)
+      const newStateSlice = selector(state)
+      if (!equalityFn(currentSlice, newStateSlice)) {
+        const previousSlice = currentSlice
+        listener((currentSlice = newStateSlice), previousSlice)
```

Comment:
```suggestion
      const nextSlice = selector(state)
      if (!equalityFn(currentSlice, nextSlice)) {
        const previousSlice = currentSlice
        listener((currentSlice = nextSlice), previousSlice)
```

Just for naming consistency.